src/coreComponents/physicsSolvers/fluidFlow/DissipationCompositionalMultiphaseFVMKernels.hpp

```diff
@@ -0,0 +1,469 @@
+                           globalIndex const rankOffset,
+                           integer const hasCapPressure,
+                           STENCILWRAPPER const & stencilWrapper,
+                           DofNumberAccessor const & dofNumberAccessor,
+                           CompFlowAccessors const & compFlowAccessors,
+                           DissCompFlowAccessors const & dissCompFlowAccessors,
+                           MultiFluidAccessors const & multiFluidAccessors,
+                           DissMultiFluidAccessors const & dissMultiFluidAccessors,
+                           CapPressureAccessors const & capPressureAccessors,
+                           PermeabilityAccessors const & permeabilityAccessors,
+                           PorosityAccessors const & porosityAccessors,
+                           RelPermAccessors const & relPermAccessors,
+                           real64 const & dt,
+                           CRSMatrixView< real64, globalIndex const > const & localMatrix,
+                           arrayView1d< real64 > const & localRhs,
+                           real64 const omega,
+                           integer const curNewton,
+                           integer const continuation,
+                           integer const miscible,
+                           real64 const kappamin,
+                           real64 const contMultiplier )
+    : Base( numPhases,
+            rankOffset,
+            hasCapPressure,
+            stencilWrapper,
+            dofNumberAccessor,
+            compFlowAccessors,
+            multiFluidAccessors,
+            capPressureAccessors,
+            permeabilityAccessors,
+            dt,
+            localMatrix,
+            localRhs ),
+    m_pres_n( dissCompFlowAccessors.get( fields::flow::pressure_n {} ) ),
+    m_compDens( dissCompFlowAccessors.get( fields::flow::globalCompDensity {} ) ),
+    m_compFrac( dissCompFlowAccessors.get( fields::flow::globalCompFraction {} ) ),
+    m_phaseRelPerm_n( relPermAccessors.get( fields::relperm::phaseRelPerm_n {} ) ),
+    m_phaseDens_n( dissMultiFluidAccessors.get( fields::multifluid::phaseDensity_n {} ) ),
+    m_phaseMassDens_n( dissMultiFluidAccessors.get( fields::multifluid::phaseMassDensity {} ) ),
+    m_phaseCompFrac_n( dissMultiFluidAccessors.get( fields::multifluid::phaseCompFraction_n {} ) ),
+    m_permeability( permeabilityAccessors.get( fields::permeability::permeability {} ) ),
+    m_dPerm_dPres( permeabilityAccessors.get( fields::permeability::dPerm_dPressure {} ) ),
+    m_omegaDBC( omega ),
+    m_curNewton( curNewton ),
+    m_continuationDBC( continuation ),
+    m_miscibleDBC( miscible ),
+    m_kappaminDBC( kappamin ),
+    m_contMultiplierDBC( contMultiplier ),
+    m_volume( dissCompFlowAccessors.get( fields::elementVolume {} ) ),
+    m_porosity_n( porosityAccessors.get( fields::porosity::porosity_n {} ) )
+  {}
+
+  struct StackVariables : public Base::StackVariables
```

Comment:
Is it really needed if none variable is added to the struct ? isn't a `using StackVariables = Base::StackVariables` possible ?